### PR TITLE
docs: update badges

### PR DESCRIPTION
**Pull request by abraham**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # Node.js ABI
 
-[![Build Status](https://travis-ci.org/lgeiger/node-abi.svg?branch=v1.0.0)](https://travis-ci.org/lgeiger/node-abi) [![Greenkeeper badge](https://badges.greenkeeper.io/lgeiger/node-abi.svg)](https://greenkeeper.io/)
+[![Build Status](https://circleci.com/gh/electron/node-abi/tree/main.svg?style=shield)](https://circleci.com/gh/electron/node-abi/?branch=main) [![Skyk badge](https://snyk.io/test/github/electron/node-abi/badge.svg)](https://snyk.io/test/github/electron/node-abi)
```
Comment (erickzhao) [inline README.md:3]:
```suggestion
[![Build Status](https://circleci.com/gh/electron/node-abi/tree/main.svg?style=shield)](https://circleci.com/gh/electron/node-abi/?branch=main) [![Snyk badge](https://snyk.io/test/github/electron/node-abi/badge.svg)](https://snyk.io/test/github/electron/node-abi)
```

Comment (malept) [reply README.md:3]: Also it'd probably be better if there was one badge per line for readability purposes.

**Review by erickzhao — APPROVED** [review]
Thank you for the catch @abraham !

**Review by dsanders11 — APPROVED** [review]
Didn't see this PR, merging in my changes from #129. Thanks for the PR @abraham, sorry it took a while to merge. 🙂

**Pull request merged**